unity/ComputerVision-UnityClient/Assets/Scripts/AppManager.cs

```diff
@@ -38,7 +43,24 @@ bool serviceAlreadyStarted
                 }
             }
         }
+        private int[][] faceDetectionRects;
+        private int[] faceRecognitionRect;
+        private static string faceRecognitionSubject;
```

Comment:
why static?

Reply:
good catch, this left from previous development.